try:
    print(x)
except:
    print("an error occured")